src/pluto/components/aggregate_report.py

```diff
@@ -0,0 +1,136 @@
+import streamlit as st
+import pandas as pd
+from numerize.numerize import numerize
+import plotly.graph_objects as go
+from src.constants import COLOR_SCHEME
+
+
+class AggregateReport:
+    def __init__(self, data, v1, v2, v3, condo, mapped):
+        self.df = data
+        self.v1 = v1
+        self.v2 = v2
+        self.v3 = v3
+        self.condo = condo
+        self.mapped = mapped
+
+    def __call__(self):
+        st.header("Aggregate Graph")
+
+        self.display_graph()
+
+        st.write(self.aggregate_df.sort_values(by="v", ascending=False))
+
+        st.info(self.info_description)
+
+    @property
+    def aggregate_df(self):
+        return self.df.loc[
+            (self.df.condo == self.condo)
+            & (self.df.mapped == self.mapped)
+            & (self.df.v.isin([self.v1, self.v2, self.v3])),
+            :,
+        ]
+
+    @property
+    def v1_records(self):
+        return self.records_by_version(self.v1)
+
+    @property
+    def v2_records(self):
+        return self.records_by_version(self.v2)
+
+    @property
+    def v3_records(self):
+        return self.records_by_version(self.v3)
+
+    def records_by_version(self, version):
+        return self.aggregate_df.loc[self.aggregate_df.v == version, :].to_dict(
+            "records"
+        )[0]
+
+    @property
+    def y_columns(self):
```

Comment:
Usually best to put these long hard-coded lists to bottom of file for readability